python/delta/utils.py

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (2020) The Delta Lake Project Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import importlib_metadata
```

Comment:
from the docs https://importlib-metadata.readthedocs.io/en/latest/ i see that there are changes in 3.7 or something... will this consistently work for all 3.x?

Reply:
thanks for clarifying. I understand that. But I am asking whether if someone runs this on python 3.8 .. will this fail? Because I am pretty sure that Apache Spark won't fail on 3.8, so we definitely don't want to create version cliffs in Delta.

Reply:
No, this does not fail on any version of Python 3, and there should be no adverse effects on 3.8+